### DISCUSSION TITLE
BACK button event

### DISCUSSION THREAD

**Discussion by moraisEltonD** [gilbarbara/react-joyride#668] [Q&A]
how can i know when the BACK button receives a click ??

**Comment by jimmythecode** [comment]
In the callback, as per: 
https://docs.react-joyride.com/callback

Line 20 ```...action === ACTIONS.PREV```

**Comment by jimmythecode** [comment]
Also, I made a tutorial that might help:
https://codesandbox.io/s/joyride-controlled-demos-hwojx

If you click "the basics" tutorial, it gives some explanation as to how all that works.

**Reply by moraisEltonD** [reply]
thank's :100:  :+1: very helpfull :1st_place_medal:
